### ISSUE TITLE
Error at step "Clone https://github.com/googlecodelabs/your-first-pwapp.git and click OK."

### ISSUE THREAD

**Issue by be-nice-do-better** [googlecodelabs/your-first-pwapp#186]
page: https://codelabs.developers.google.com/codelabs/your-first-pwapp/#1
When perform step: 4. Clone https://github.com/googlecodelabs/your-first-pwapp.git and click OK.

glitch.com says:
An error occurred when trying to create this project, please try again later.

**Comment by jjj257** [comment]
I have the same problem..

**Comment by nerolee7** [comment]
Same

**Comment by petele** [comment]
I think Glitch.com might be having some issues, you can try using the alternate method, of downloading and running locally.

**Comment by petele** [comment]
From: https://support.glitch.com/t/unable-to-create-new-project-by-cloning-repository/13324

> in the meantime, if you create a new project with either of the other options, you can then import the googlecodelabs/your-first-pwapp repo in the editor and that should work: within the editor, on the lower right corner you’ll see a “Tools” button. in that menu, select “Git, Import, Export” and from there you should be able to import this repo.

**Comment by slbccfl** [comment]
I still get a similar error message when using the import after creating a new project Glitch.

**Comment by petele** [comment]
I was able to get it to work by creating a new "hello-express" project, then clicking tools -> git import & export -> import from GitHub -> googlecodelabs/your-first-pwapp

Not ideal I know, hopefully they'll get stuff fixed shortly. Sorry for the inconvenience!

**Comment by slbccfl** [comment]
Using "hello-express" instead of "hello-webpage" was slightly different. It gives to a message "Importing from https://github.com/googlecodelabs/your-first-pwapp.git" but just hangs until eventually "Page unresponsive" pop-up comes up.  I'm starting to wonder about what kind of problems Glitch is having.

**Comment by be-nice-do-better** [comment]
I temporarily fixed this by the below two steps:
1. fork https://github.com/googlecodelabs/your-first-pwapp to my own repo https://github.com/jw-test/your-first-pwapp
2. use **https://github.com/jw-test/your-first-pwapp.git** instead of https://github.com/googlecodelabs/your-first-pwapp.git on glitch.com

**Comment by slbccfl** [comment]
I finally found what I was doing wrong: I was using the "clone or download" button in Github and then copying the entire URL:  https://github.com/googlecodelabs/your-first-pwapp.git when trying to import into/over the "hello-express" template. 
The option to use the "New Project"/"Clone from Git Repo" from your portfolio page prompts for "https://github.com/orgname/reponame.git" and still hangs if I use that. 
Starting a "New Project" and choosing the "hello-express" template and then using "Tools"/"Git Import and Export"/"Import from GitHub" prompts for "user/repo:path". I was entering the entire URL when I used this approach. When I enter only "googlecodelabs/your-first-pwapp" (not the entire URL) this import works.
